<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>config::AddSensorComboDialog</class>
 <widget class="QDialog" name="config::AddSensorComboDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Sensor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="textFieldsLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <spacer name="verticalSpacer_top">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="SensorLayout">
       <item>
        <widget class="QLabel" name="SensorLabel">
         <property name="text">
          <string>Sensor Name</string>
         </property>
         <property name="buddy">
          <cstring>SensorBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="SensorBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="DeviceLayout">
       <item>
        <widget class="QLabel" name="DeviceLabel">
         <property name="text">
          <string>Device</string>
         </property>
         <property name="buddy">
          <cstring>DeviceText</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DeviceText">
         <property name="inputMask">
          <string notr="true"/>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="IdLayout">
       <item>
        <widget class="QLabel" name="IdLabel">
         <property name="text">
          <string>Sensor Id</string>
         </property>
         <property name="buddy">
          <cstring>IdText</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IdText">
         <property name="inputMask">
          <string notr="true"/>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_bottom">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>SensorBox</tabstop>
  <tabstop>DeviceText</tabstop>
  <tabstop>IdText</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>config::AddSensorComboDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>config::AddSensorComboDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
